packages/notebook/style/notebookfooter.css

```
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+.jp-Notebook-footer {
+  height: 10%;
```

Could you set a fix height because this make the area height change with the window size?

```suggestion
  height: ...px;
```